airbyte-db/lib/src/main/java/io/airbyte/db/check/DatabaseAvailabilityCheck.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.db.check;
+
+import static org.jooq.impl.DSL.select;
+
+import io.airbyte.commons.lang.Exceptions;
+import io.airbyte.db.Database;
+import java.util.Optional;
+import java.util.function.Function;
+import org.jooq.DSLContext;
+import org.slf4j.Logger;
+
+/**
+ * Performs a check to verify that the configured database is available.
+ */
+public interface DatabaseAvailabilityCheck extends DatabaseCheck {
+
+  /**
+   * The number of times to check if the database is available. TODO replace with a default value in a
+   * value injection annotation
+   */
+  int NUM_POLL_TIMES = 10;
```

Comment:
This seems like a constant, and therefore should be `final` (maybe not super important given the comment above saying that this will be replaced)

Reply:
Good catch.  This was copy/pasted from the other classes, so its probably also wrong there.  Will fix.

Reply:
This is a constant in an interface, so marking it `final` doesn't do anything.

Reply:
Whoops, excuse my tunnel vision here 🤦